sharded_queue/api.lua

```diff
@@ -92,6 +92,12 @@ function sharded_tube.take(self, timeout, options)
     end
     options.tube_name = self.tube_name
 
+    local wait_factor = 1
```

Comment:
Default value should not be 1, or it won't `fix` anything by default

Reply:
Default value is 2.